docs/.link-check-errors.txt

```diff
@@ -2,5 +2,6 @@ Broken links found by `yarn docs:link-check` that exist:
 
 - https://mui.com/blog/material-ui-v4-is-out/#premium-themes-store-✨
 - https://mui.com/size-snapshot
+- https://mui.com/x/introduction/licensing/#pro-premium
```

Comment:
Does this page even exist?

Reply:
It was generated by `yarn docs:link-check`

Reply:
Does it add it back if you remove it and run `yarn docs:link-check` again?